gsoc/2021/posts/trung_ngo_tan/post04.md

```diff
@@ -0,0 +1,48 @@
+# GSoC'21 RoboComp project: Sign language recognition
+
+29th July 2021
+
+## Installation:
+Both components are runned on Python3 environment. In order to ease the pain of installation, I use Anaconda virtual enviroment. Link: https://docs.anaconda.com/anaconda/install/index.html
+
+The requirement libraries for running components:
++ cudatoolkit == 11.0.2
++ cudnn == 8.0.5
++ mediapipe == 0.8.x
++ opencv == 4.5.2
++ pyside2 
++ onnxruntime-gpu == 1.8.1 (should install with pip)
++ tensorRT == 8.0.1 (Should install with pip)
```

Comment:
It looks like some of the packages (like Torch) are missing from this list. Can you add those as well?